pkgs/applications/misc/1password/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchzip, makeWrapper }:
+
+let
+  src =
+    if stdenv.system == "i686-linux" then fetchzip {
+      url = "https://cache.agilebits.com/dist/1P/op/pkg/v0.4/op_linux_386_v0.4.zip";
+      sha256 = "0mhlqvd3az50gnfil0xlq10855v3bg7yb05j6ndg4h2c551jrq41";
+      stripRoot = false;
+    } else fetchzip {
+      url = "https://cache.agilebits.com/dist/1P/op/pkg/v0.4/op_linux_amd64_v0.4.zip";
+      sha256 = "15cv8xi4slid9jicdmc5xx2r9ag63wcx1mn7hcgzxbxbhyrvwhyf";
+      stripRoot = false;
+    };
+in stdenv.mkDerivation {
+  name = "1password-0.4";
+  inherit src;
+  nativeBuildInputs = [ makeWrapper ];
+  installPhase = ''
+    mkdir -p $out/bin
+    install -D op $out/share/1password/op
+
+    # https://github.com/NixOS/patchelf/issues/66#issuecomment-267743051
+    makeWrapper $(cat $NIX_CC/nix-support/dynamic-linker) $out/bin/op \
+      --argv0 op \
```

Comment:
Sometime programs are confused if argv0 is ld-linux-x86-64.so.2 instead of the original executable name.
So better be save then sorry.